packit_service_fedmsg/callbacks.py

```diff
@@ -226,6 +226,12 @@ def _openscanhub_task_started(
     )
 
 
+def _logdetective_analysis_result(topic: str, event: dict, packit_user: str):
```

Comment:
Nitpick - for consistency with the rest of the callbacks please add return value annotation (` -> CallbackResult:`)